src/zoid/fields/component.jsx

```diff
@@ -117,6 +117,11 @@ export function getFieldsComponent() : FieldsComponent {
                     value:      getEnv
                 },
 
+                onMount: {
```

Comment:
In other components we use `onInit` for this. Can we follow the same convention here?